api/src/test/java/com/graphhopper/util/HelperTest.java

```diff
@@ -55,6 +55,8 @@ public void testRound() {
         assertEquals(100.94, Helper.round(100.94, 2), 1e-7);
         assertEquals(100.9, Helper.round(100.94, 1), 1e-7);
         assertEquals(101.0, Helper.round(100.95, 1), 1e-7);
+        // todonow
+        assertEquals(1040.02, Helper.round(1040.02125359, -2), 1.e-7);
```

Comment:
I ran into this while working on the example? How is this meant to work? Currently `Helper.round(1040.02125359, -2)` returns `1000`.

Reply:
Ah now I get it: `-2` means the decimal point is shifted to the *left* (by two). Rather counter-intuitive if you ask me because setting the exponent to `-2` I would expect the precision is `10^-2`, not `10^2`.

Reply:
Hmmh, not sure why I did it that way for Helper. Should we can change it?

Reply:
Hm for me its fine like this (now that I know it :smile: ). Rounding to `2` means 'two decimal places', from this point of view it makes sense?

Reply:
Yes, makes a bit sense :)